plugin/src/main/java/com/denizenscript/denizen/objects/properties/material/MaterialLevel.java

```diff
@@ -50,41 +51,67 @@ public static void registerTags() {
         // @returns ElementTag(Number)
         // @group properties
         // @description
-        // Returns the maximum level for a levelable material (like water, lava, and Cauldrons), a cake, or a beehive.
+        // Returns the maximum level for a valid material.
+        // Valid materials include water, lava, cauldrons, cake, beehives, and snow.
         // -->
         PropertyParser.<MaterialLevel>registerTag("maximum_level", (attribute, material) -> {
             return new ElementTag(material.getMax());
         });
 
+        // <--[tag]
+        // @attribute <MaterialTag.minimum_level>
+        // @returns ElementTag(Number)
+        // @group properties
+        // @description
+        // Returns the minimum level for a valid material.
+        // Valid materials include water, lava, cauldrons, cake, beehives, and snow.

```

Comment:
This meta should clarify that there's only a non-zero value for snow (if it even is actually non-zero for snow?)